langchain/chains/llm_math/base.py

```diff
@@ -68,6 +68,35 @@ def _process_llm_result(self, t: str) -> Dict[str, str]:
             raise ValueError(f"unknown format from LLM: {t}")
         return {self.output_key: answer}
 
+    async def _aprocess_llm_result(self, t: str) -> Dict[str, str]:
+        python_executor = PythonREPL()
+        if self.callback_manager.is_async:
```

Comment:
Remind me, we have a plan to remove this branching, and allow only async callback managers with async methods, yes?

Reply:
That's correct